### DISCUSSION TITLE
I am running a Hubs cloud instance on AWS and it keeps crashing.

### DISCUSSION THREAD

**Discussion by k888885** [mozilla/hubs#3892] [Hubs Cloud Questions] [answered]
**Description**
It crashes during meetings whit more than 3 people in the room. The crashes seem to be somehow correlated with the number of media objects in the room. Sometimes the crash with only 2-3 videos-cameras in the room while otherwise, it can withstand 8-10 such objects. After the crash, the cloud instance is offline from 10-20 minutes.
Why does this keep happening and how can I troubleshoot?

**To Reproduce**
Add multiple multimedia materials to the scene.

**Expected behavior**
The Hubs Cloud crashes and Hubs cloud is offline page appears when refreshing

**Hardware**
 - Device: [Desktop]
 - OS: [Windows]
 - Browser: [Firefox, Chrome]

**Comment by robinkwilson** [comment]
Hello! What instance size are you using? The crashing sounds like you're using a `t3.micro` is that correct? 

If it is a `t3.micro`, I recommend scaling up to a `t3.small` or a `t3.medium`.

**Comment by k888885** [comment]
No, I am using t3.large. I have tried other instances like c4Xlarge to test if
the stack is running out of memory but the crash happens regardless. I have done
a ton of testing like removing 360 video or geometry but I can not fix the
issue. I have even deleted the scene in Spoke and made an identical one but even
that did not work. Also, I have tried terminating the EC2 instance but I don't
think that this will solve the problem.
Also, I have watched all of your Youtube videos about hubs  cloud setup multiple
times so it is very unlikely that I have missed anything that Is mentioned
there.
I am using the personal plan could this be a problem?

**Reply by robinkwilson** [reply]
Can you tell me more information about your deployment?

Including: 
- Scene size / Poly count
- Number of videos
- Types of videos
- Any relevant content I missed asking about
- Actions you've taken in the Admin Panel
- Actions you've taken in AWS to affect the EC2 resources
- Anything else that seems relevant

The personal plan should be working well especially with a t3.large. I'm surprised by this behavior.

**Comment by robinkwilson** [comment, marked as answer]
@k888885 I noticed in discord you mentioned using 4K textures and a community user volunteered to help shrink those down (since those large textures crashed for them in the past too)! Hopefully this is resolved! 

Is this correct and is this resolved?

**Reply by k888885** [reply]
Textures were the problem I have shrunk everything down to 2048 and now it works great. The Skybox image is now far from optimal but we have made a stress test with 30+ videos and the server did not crash :D Thanks for the help!

**Reply by robinkwilson** [reply]
Wonderful!!! So glad you were able to get the help you needed and it's resolved! :D
